content_scripts/link_hints.coffee

```diff
@@ -167,11 +167,25 @@ class LinkHintsMode
         element.getAttribute("aria-disabled")?.toLowerCase() in ["", "true"])
       return [] # This element should never have a link hint.
 
+    # checks for every valid version of ng-click
```

Comment:
This comment should mention something about Angular.js so we know *why* we're doing it, as well as what we're doing.